app_monitoring/kube/pinpoint-template-ephemetal.yaml

```diff
@@ -0,0 +1,159 @@
+apiVersion: v1
+kind: Template
+labels:
+  template: pinpoint-apm-ephemeral
+message: Pinpoint APM Ephemeral
+metadata:
+  annotations:
+    description: Pinpoint APM Ephemeral
+    iconClass: icon-jenkins
+    tags: pinpoint
+  name: pinpoint-ephemeral-template
+  namespace: openshift
+objects:
+- apiVersion: v1
+  kind: ServiceAccount
+  metadata:
+    name: pinpoint
+- apiVersion: "v1"
+  kind: "ImageStream"
+  metadata:
+    name: "pinpoint"
+- apiVersion: v1
+  kind: Service
+  metadata:
+    name: pinpoint-apm
+  spec:
+    ports:
+    - name: pinpoint
+      port: 28080
+      protocol: TCP
+      targetPort: 28080
+    selector:
+      name: pinpoint-apm
+    sessionAffinity: None
+    type: ClusterIP
+  status:
+    loadBalancer: {}
+- apiVersion: v1
+  kind: Service
+  metadata:
+    name: pinpoint-testapp
+  spec:
+    ports:
+    - name: testapp
+      port: 28081
+      protocol: TCP
+      targetPort: 28081
+    selector:
+      name: pinpoint-apm
+    sessionAffinity: None
+    type: ClusterIP
+  status:
+    loadBalancer: {}
+- apiVersion: v1
+  kind: Service
+  metadata:
+    name: pinpoint-collector
+  spec:
+    ports:
+    - name: pinpoint-tcp
+      port: 9994
+      protocol: TCP
+      targetPort: 9994
+    - name: pinpoint-udp1
+      port: 9995
+      protocol: UDP
+      targetPort: 9995
+    - name: pinpoint-udp12
+      port: 9996
+      protocol: UDP
+      targetPort: 9996
+    selector:
+      app: pinpoint-apm
+    sessionAffinity: None
+    type: ClusterIP
+  status:
+    loadBalancer: {}
+- apiVersion: v1
+  kind: Route
+  metadata:
+    name: webinterface
+  spec:
+    to:
+      kind: Service
+      name: pinpoint-apm
+- apiVersion: v1
+  kind: Route
+  metadata:
+    name: testapp
+  spec:
+    to:
+      kind: Service
+      name: pinpoint-testapp
+- apiVersion: "v1"
+  kind: "BuildConfig"
+  metadata:
+    name: "pinpoint"
+  spec:
+    triggers:
+      - type: ConfigChange
+      - type: ImageChange
+    source:
+      type: Git
+      git:
+        uri: https://github.com/redhat-cop/containers-quickstarts.git
+    strategy:
+      type: "Docker"
+      dockerStrategy:
+        dockerfilePath: app_monitoring/
```

Comment:
There should be a section on how to interact with the sample application. Otherwise, no values ever get populated in the APM tool. Be sure to describe each action the user should take to generate the visuals below